mne/data/coil_def.dat

```diff
@@ -501,3 +501,72 @@
  -0.1250  4.082e-03 -7.071e-03  5.000e-02  0.000  0.000  1.000
  -0.1250 -4.082e-03  7.071e-03  5.000e-02  0.000  0.000  1.000
  -0.1250 -4.082e-03 -7.071e-03  5.000e-02  0.000  0.000  1.000
+2  10001    0   2  1.4859e-02  5.7404e-02	"Artemis123 axial gradiometer size = 14.8590  mm base = 57.4040  mm"
+  1.0000    0.000e+00     0.000e+00     0.000e+00     0.000e+00     0.000e+00     1.000e+00
+ -1.0000    0.000e+00     0.000e+00     5.740e-02     0.000e+00     0.000e+00     1.000e+00
+2  10001    1   8  1.4859e-02  5.7404e-02	"Artemis123 axial gradiometer size = 14.8590  mm base = 57.4040  mm"
+  0.2500    3.715e-03     3.715e-03     0.000e+00     0.000e+00     0.000e+00     1.000e+00
+  0.2500   -3.715e-03     3.715e-03     0.000e+00     0.000e+00     0.000e+00     1.000e+00
+  0.2500   -3.715e-03    -3.715e-03     0.000e+00     0.000e+00     0.000e+00     1.000e+00
+  0.2500    3.715e-03    -3.715e-03     0.000e+00     0.000e+00     0.000e+00     1.000e+00
+ -0.2500    3.715e-03     3.715e-03     5.740e-02     0.000e+00     0.000e+00     1.000e+00
+ -0.2500   -3.715e-03     3.715e-03     5.740e-02     0.000e+00     0.000e+00     1.000e+00
+ -0.2500   -3.715e-03    -3.715e-03     5.740e-02     0.000e+00     0.000e+00     1.000e+00
+ -0.2500    3.715e-03    -3.715e-03     5.740e-02     0.000e+00     0.000e+00     1.000e+00
+2  10001    2  14  1.4859e-02  5.7404e-02	"Artemis123 axial gradiometer size = 14.8590  mm base = 57.4040  mm"
+  0.2500    0.000e+00     0.000e+00     0.000e+00     0.000e+00     0.000e+00     1.000e+00
+  0.1250    6.066e-03     0.000e+00     0.000e+00     0.000e+00     0.000e+00     1.000e+00
+  0.1250   -6.066e-03     7.429e-19     0.000e+00     0.000e+00     0.000e+00     1.000e+00
```

Comment:
That looks super useful. Maybe just add a one liner to replace any values in the `int_points` list that are <1e-10 with zero, or something similar.